src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -176,19 +172,19 @@ public String getDataInventory(String params) {
     }
 
     public String getReloadToken(InputStream input) {
-        try (BufferedReader br = new BufferedReader(new InputStreamReader(input))) {
-            return br.lines()
-                    .map(RELOAD_TOKEN_PATTERN::matcher)
-                    .filter(Matcher::find)
-                    .map(m -> m.group(1))
-                    .findFirst().orElse(null);
-
-        } catch (IOException e){
+        try {
+            return getReloadToken(IOUtils.read(input));
+        } catch (IOException e) {
             e.printStackTrace();
             return null;
         }
```

Comment:
This is replacing a buffered reader that will read lines one by one and short-circuit when one matches, with a full download of the whole page to one massive string, and then doing a regex search over that huge string.

Keep it as it was